src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -341,27 +341,104 @@ export class PatientViewPageStore
                     sampleIdToTumorType[mutation.sampleId]);
             }), "id");
 
-            return new Promise((resolve, reject) => {
-                const oncokbSearchPromise = oncokbClient.searchPostUsingPOST(
-                    {body: generateEvidenceQuery(queryVariants)});
+            const oncokbSearchPromise = oncokbClient.searchPostUsingPOST(
+                {body: generateEvidenceQuery(queryVariants)});
+
+            const evidenceLookupPromise = oncokbClient.evidencesLookupPostUsingPOST(
+                {body: generateEvidenceQuery(queryVariants)});
 
+            return new Promise((resolve, reject) => {
                 // TODO return type is not correct for the auto-generated API!
                 // generated return type is Array<IndicatorQueryResp>,
                 // but the actual return type is {meta: {} data: Array<IndicatorQueryResp>}
                 // that's why here we need to force data type to be any and get actual data by data.data
-                oncokbSearchPromise.then((data:any) => {
+                Promise.all([oncokbSearchPromise, evidenceLookupPromise]).then((values) => {
                     const oncoKbData:IOncoKbData = {
                         sampleToTumorMap: sampleIdToTumorType,
-                        indicatorMap: generateIdToIndicatorMap(data.data)
+                        indicatorMap: generateIdToIndicatorMap((values[0] as any).data),

```

Comment:
Instead of letting the `any` propagate throughout the whole function, either fix the generated API code to have the correct type or cast the promise as the correct type. You can do `as any as CorrectType`.